backend/src/packages/meditation/meditation.controller.ts

```diff
@@ -0,0 +1,98 @@
+import { APIPath } from '#libs/enums/enums.js';
+import {
+  type APIHandlerOptions,
+  type APIHandlerResponse,
+  BaseController,
+} from '#libs/packages/controller/controller.js';
+import { HTTPCode } from '#libs/packages/http/http.js';
+import { type Logger } from '#libs/packages/logger/logger.js';
+
+import { MeditationApiPath } from './libs/enums/enums.js';
+import { type MeditationEntryCreateRequestDto } from './libs/types/types.js';
+import { type MeditationService } from './meditation.service.js';
+
+/**
+ * @swagger
+ * components:
+ *    schemas:
+ *      MeditationEntryRequest:
+ *        type: object
+ *        properties:
+ *          topicName:
+ *            type: string
+ *          audioUrl:
+ *            type: string
+ *            format: url
+ *      MeditationEntryResponse:
+ *        type: object
+ *        properties:
+ *          topicName:
+ *            type: string
+ *          audioUrl:
+ *            type: string
+ *            format: url
+ *          createdAt:
+ *            type: string
+ *            format: date-time
+ *          updatedAt:
+ *            type: string
+ *            format: date-time
+ */
+class MeditationController extends BaseController {
+  private meditationService: MeditationService;
+
+  public constructor(logger: Logger, meditationService: MeditationService) {
+    super(logger, APIPath.MEDITATION);
+
+    this.meditationService = meditationService;
+
+    this.addRoute({
+      path: MeditationApiPath.ROOT,
+      method: 'POST',
+      handler: (options) => {
```

Comment:
we should add validation for body here as in auth controller

Reply:
Done